src/Dinkbit/Notifyme/Gateways/HipChat.php

```diff
@@ -193,13 +189,17 @@ public function jsonError($rawResponse)
      *
      * @param string $string
      *
-     * @return bool
+     * @return bool|array
      */
     protected function isJson($string)
```

Comment:
and doesn't json_decode return null on error anyway, so this entire method is a bit over kill